packages/amplication-client/src/Project/ProjectPage.scss

```diff
@@ -4,4 +4,21 @@
   display: flex;
   height: 100%;
   flex: 1 auto;
+
+  .commit-page-empty-state {
+    display: flex;
+    flex-direction: column;
+    position: absolute;
+    left: 44%;
+    top: 28%;
```

Comment:
why are we using these excat values?

Reply:
yes. I used position relative on the parent and absolute on the child and I need these exact values

Reply:
44% may work, but it is not accurate.  especially on larger or smaller screens then the one you used. 
There are a couple of other ways to achieve it - 
If you want to use %, to be more accurate, you can use -  left : calc(50% - $image-width/2)
but a better and easier way to do it is to use flex with align center. 
This way is built proof for any change in the image size and does not require you to deal with %. 
also, it resolve both width and height

```css 
    display: flex;
    align-items: center;
    justify-content: center;
    height: 100%;
    width: 100%;
```

Reply:
I will try it. Thanks